### ISSUE TITLE
Add flash attention to requirements.txt

### ISSUE THREAD

**Issue by Anthonyg5005** [turboderp/exllamav2#367]
I'd send in a PR but I wouldn't want to break anything.
I was wondering why nothing was happening after reinstall of windows until I realized flash attention wasn't in the requirements. after installing flash attention from [text-generation-webui/requirements.txt](https://github.com/oobabooga/text-generation-webui/blob/1934cb61ef879815644277c01c7295acbae542d8/requirements.txt#L63) I was able to get it to work and start converting.
Only problem I see is having to update it on every flash attention release. Another solution could be just adding it to the README instructions.

Love your work turbo.

**Comment by turboderp** [comment]
It's supposed to be able to run without flash-attn, which is why it isn't in the requirements. Could be something broke and I didn't notice?

**Comment by Anthonyg5005** [comment]
I'm not really sure. I might've not waited long enough. I gave it like 5 minutes.
To be more specific I was quantizing a model fine-tuned on mistral 7b v0.1
I have a 12GB 3060, Driver Version: 551.61,  CUDA Version: 12.4
Windows 11 Build 22631
Python 11, Torch 2.2.1+cu121
I only started converting after installing flash attention.
I also just installed windows temporarily while I try to fix my main drive. Installed VS 2019 and added the bin with cl.exe to path
